renderer/components/Directory.jsx

```diff
@@ -1,76 +1,94 @@
-import React from 'react';
+import React, { Component } from 'react';
+import cx from 'classnames';
 import File from './File.jsx';
 import CreateMenu from './CreateMenu.jsx';
 import CreateForm from './CreateForm.jsx';
 import RenameForm from './RenameForm.jsx';
 
-export default class Directory extends React.Component {
-  constructor() {
-    super();
-  }
+export default class Directory extends Component {
+  handleListItemClick = proxyEvent => (
```

Comment:
What about code consistency - I've refactored all other files except App.jsx already. It will be next step